openforcefield/topology/molecule.py

```diff
@@ -1913,24 +1916,68 @@ def generate_conformers(self,
 
         """
         if isinstance(toolkit_registry, ToolkitRegistry):
-            return toolkit_registry.call('generate_conformers', self,
-                                         clear_existing)
+            return toolkit_registry.call('generate_conformers', self, num_conformers=num_conformers,
+                                         clear_existing=clear_existing)
         elif isinstance(toolkit_registry, ToolkitWrapper):
             toolkit = toolkit_registry
-            return toolkit.generate_conformers(self, clear_existing)
+            return toolkit.generate_conformers(self, num_conformers=num_conformers, clear_existing=clear_existing)
         else:
             raise InvalidToolkitError(
                 'Invalid toolkit_registry passed to generate_conformers. Expected ToolkitRegistry or ToolkitWrapper. Got  {}'
                 .format(type(toolkit_registry)))
 
+    def compute_partial_charges_am1bcc(self, toolkit_registry=GLOBAL_TOOLKIT_REGISTRY):
+        """
+        Calculate partial atomic charges for this molecule using an underlying toolkit
+
+        Parameters
+        ----------
+        toolkit_registry : openforcefield.utils.toolkits.ToolRegistry or openforcefield.utils.toolkits.ToolkitWrapper, optional, default=None
+            :class:`ToolkitRegistry` or :class:`ToolkitWrapper` to use for SMILES-to-molecule conversion
+
+        Examples
+        --------
+
+        >>> molecule = Molecule.from_smiles('CCCCCC')
+        >>> molecule.generate_conformers()
+        >>> molecule.compute_partial_charges()
+
+        Raises
+        ------
+        InvalidToolkitError
+            If an invalid object is passed as the toolkit_registry parameter
+
+        """
+        if isinstance(toolkit_registry, ToolkitRegistry):
+            charges = toolkit_registry.call(
+                      'compute_partial_charges_am1bcc',
+                      self
+            )
+        elif isinstance(toolkit_registry, ToolkitWrapper):
+            toolkit = toolkit_registry
+            charges = toolkit.compute_partial_charges_am1bcc(
```

Comment:
Isn't the spacing a little silly and unnecessarily verbose here? This could just be 
```python
        elif isinstance(toolkit_registry, ToolkitWrapper):
            charges = toolkit_registry.compute_partial_charges_am1bcc(self)
```

Reply:
Yeah, `yapf` made some interesting decisions about where to break lines. I'm treating these on a one-by-one basis as I find them. I'll go ahead and clean this one up in the branch.

Reply:
Fixed in PR